ui/src/components/User/ChangePassword.vue

```diff
@@ -135,6 +138,17 @@ const showNewPassword = ref(false);
 const showConfirmPassword = ref(false);
 const { name, email, username, recoveryEmail } = authStore;
 
+const handleNewPasswordChange = () => {
+  if (!newPasswordConfirm.value || !newPassword.value) return;
```

Comment:
We should only consider it empty if both are empty or if both are equal to each other. In the way that it is now, if any of them is empty and I try to change the password, it is not going to identify that they are not equal.